docs/information/manifest.md

```diff
@@ -70,6 +70,7 @@ All properties for extension or command-specific preferences. Use the [Preferenc
 | Property                                      | Description                                                                                                                                                                                                                |
 | --------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | name<mark style="color:red;">\*</mark>        | A unique id for the preference.                                                                                                                                                                                            |
+| title<mark style="color:red;">\*</mark>       | The display name of the preference, shown to the user in Raycast.                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I think we should move the title from `checkbox` additional property to here:

https://github.com/raycast/extensions/blob/9f4cf2803678ea15b84e4e13b31bfa454c32a843/docs/information/manifest.md?plain=1#L85

What do you think?

Reply:
I didn’t notice it was there. I agree it should be moved. I will copy the content from the checkbox section here and then delete the checkbox section. Is that okay?

Reply:
Yes that's what I was thinking too.

Reply:
Thank, I updated it.

Reply:
For sure, you can resolve this comment now. I approved the changes.

Reply:
Thank you for your support!!